contents/android/IntroToAndroid.md

```diff
@@ -0,0 +1,145 @@
+# Introduction to Android App Development
+
+Authors: [Lee Yan Hwa](https://github.com/leeyh20)
+
+- [Getting Started](#1-getting-started)
+    - [What is Android?](#11-what-is-android)
+    - [Develop your first app today!](#12-develop-your-first-app-today)
+
+- [Android is not magic](#2-android-is-not-magic)
+    - [App Components](#21-app-components)
+    - [Activity Lifecycle](#22-activity-lifecycle)
+    - [The Architecture of Android](#23-the-architecture-of-android)
+    - [Resources](#24-resources)
+    - [Testing](#25-testing)
+- [Why should I make native Android apps?](#3-why-should-i-make-native-android-apps)
+- [Further Readings](#4-further-readings)
+
+## 1. Getting Started
+
+### 1.1 What is Android?
+
+Android is a Linux-based mobile phone operating system (OS), created by Android Inc. which was then acquired by Google. It was launched in 2008. Not only does it power phones, it is already the OS of watches, televisions and even car stereo systems.
```

Comment:
Minimize info that may not be of interest to the reader. `launched in 2008` may be relevant to show the OS is mature but `Android inc` may not be so.